insights/specs/datasources/lpstat.py

```diff
@@ -37,7 +37,7 @@ def lpstat_protocol_printers_info(broker):
         for line in content:
             if "device for " in line:
                 "Remove printer address information"
-                result.append(line.rsplit(":", 1)[0])
+                result.append(":".join(line.split(":", 2)[0: 2]) if '://' in line else line)
```

Comment:
@wushiqinlou looking at this line for me it is hard to understand what exactly is happening, purely form readability standpoint. What do you think about logic like this?
```python
result.append(line.split("://", 1)[0] if '://' in line else line)
```

Reply:
@vdymna - Updated, please have a check.

Reply:
Looks good to me, @xiangce are you good with this approach?

Reply:
@vdymna - Good to me, thanks!